lib/downloadManager.js

```diff
@@ -1,7 +1,6 @@
 const fs = require('fs');
 const path = require('path');
 const readline = require('readline');
-const http = require('http');
 const request = require('request');
```

Comment:
https://www.npmjs.com/package/request
This package was deprecated. Replace to another package.

Reply:
done